rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/widget/OneScanCovidRDTBarcodeFactoryTest.java

```diff
@@ -14,10 +14,11 @@
 import io.ona.rdt.application.RDTApplication;
 import io.ona.rdt.fragment.RDTJsonFormFragment;
 import io.ona.rdt.widget.OneScanCovidRDTBarcodeFactory;
+import io.ona.rdt.widget.RDTBarcodeFactory;
 
 public class OneScanCovidRDTBarcodeFactoryTest extends WidgetFactoryRobolectricTest {
 
-    private OneScanCovidRDTBarcodeFactory oneScanCovidRDTBarcodeFactory;
+    private RDTBarcodeFactory oneScanCovidRDTBarcodeFactory;
```

Comment:
Any reason for this change?

Reply:
actually, this test passed locally but failed in the pipeline so I tried to do this but the test still failed in the pipeline. Now I'm reverting the change.